src/OrchardCore/OrchardCore.ContentManagement.Display/ViewModels/ContentPartViewModel.cs

```diff
@@ -15,4 +18,21 @@ public ContentPartViewModel(ContentPart contentPart)
 
         public ContentPart ContentPart { get; set; }
     }
+
+    // Models an explicit part (part name != content type name)
+    public class ExplicitContentPartViewModel : Shape
+    {
+        public ExplicitContentPartViewModel()
+        {
+        }
```

Comment:
remove

Reply:
I was copying the fact that ContentPartViewModel had an explicit parameterless constructor for consistency. I've removed both of those now